tests/Integration/Options/get.php

```diff
@@ -1,16 +1,38 @@
 <?php
 
-namespace WPMedia\Options\Tests\Integration\Options;
+namespace WPMedia\Options\Tests\Unit\Options;
 
-use WPMedia\Options\Tests\Integration\TestCase;
+use WP_Rocket\Admin\Options;
+use WPMedia\Options\Tests\Unit\TestCase;
```

Comment:
Is this on purpose? we're in the integration tests for the `get` method

Reply:
Doh. Fixed in https://github.com/wp-media/options/pull/1/commits/c13c23d2a8652d15273306dec4eb5d73e547ac68.